### DISCUSSION TITLE
一些 ideas

### DISCUSSION THREAD

**Discussion by SinglePoi** [cuixueshe/earthworm#334] [Ideas]
#### 多人组队
这样或许会需要额外的排行版或分享海报，存在拉新的可能
#### 随机刷题
随机选取课程的 100 个句子，完成即可获取积分(?)
#### 定级
单纯的排行版无法凸显卷王的气质，唯有顶级的级别才能彰显卷王气息
取一个时间段作为一个赛季，通过刷题量完成晋级，赛季结束或许可以获得徽章(?)
级别类似 青铜 黑铁 白银 黄金 砖石 等

**Comment by hildxd** [comment]
好 你来安排下？我来辅助

**Reply by SinglePoi** [reply]
我不会（理直气壮
